about.html

```diff
@@ -0,0 +1,105 @@
+    <div class="about_desc">
+        <p>Welcome to Music Teaching Online, a company conceived through the frustrations of not being able to find a local music teacher for our kids, or when we did find a teacher,
+            who was often some distance away, lessons were missed due to traffic, delayed meetings or bad weather!</p>
+        <p class="about_desc_desk">The founder of Music Teaching Online, who has over 25 years of experience as a music
+            educator, initially set up an online help <br>group to provide students and parents just like us with support, but as demand significantly grew and technology advanced,
+            we decided to share this help group to a wider audience and make it easier for you to access fantastic music teachers, not only locally but worldwide.</p>
+    </div>
+    </div>
+    <p class="about_contact">Please contact us per Email for any further questions about BA Sounds!<br><a href="https://basounds2020@gmail.com" class="email_link">basounds2020@gmail.com</a> </p>
+</header>
+<section class="section_two">
+<h2>BA Sounds Logo</h2>
+    <span class="title_line logo_line"></span>
+    <p>The logo of BA Sounds was decided through the logo competition from June 8 to June 7.<br>
+    <span class="logo_desc">After online public voting and Selection Committee's judging process, the logo from Naresh Agrewal won the prize.
+    After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:</span></p>
+    <img src="assets/ba_logo.png" alt="BA Sounds' logo">
+</section>
+<span class="divider"></span>
+<section class="section_three">
+    <h3>See previous concerts by <br>BA Sounds</h3>
+    <span class="title_line logo_line"></span>
+    <p>Take a look at the last two concerts that took place in Brooklyn, NY and Chicago, Illanois.</p>
+    <div class="concert_contain">
+    <div class="concert part1"><h2>2022</h2><h3>BA Sounds concert in Brooklyn, NY.</h3></div>
+    <div class="concert part2"><h2>2021</h2><h3>BA Sounds concert in Chicago, Illanois.</h3></div>
+    </div>
+</section>
+<section class="partner">
+    <h2 class="partner_title">Partners</h2>
+    <span class="title_line"></span>
+    <div class="logo_list">
+        <a href="https://apple.com"><i class="fa-brands fa-apple"></i></a>
+        <a href="https://spotify.com"><i class="fa-brands fa-spotify"></i></a>
+        <a href="https://play-goole.com"><i class="fa-brands fa-google-play"></i></a>
+        <a href="https://discord.com"><i class="fa-brands fa-discord"></i></a>
+        <a href="https://vimeo.com"><i class="fa-brands fa-vimeo-v"></i></a>
+    </div>
+</section>
+<footer>
+    <div class="foot_content">
+        <div class="left_section">
+            <h3>BA Sounds<br> Learn Music Online</h3>
+        </div>
+        <div class="right_section">
+            <p>2022 BA Sounds Ethiopia.<br class="no_break"> &copy;Some Rights Reserved</p>
+        </div>
+    </div>
+</footer>
```

Comment:
- [ ] Your about section footer is great, but on the desktop view of the template, the 'Partners' section is not being displayed and the footer has inverted colors, a dark background, and white text. Kindly change that so your 'About page' footer can look like the template.
 - Your design:
 - 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/61919617/203640002-cc3f4f65-5d33-48c9-92db-181c33fe73a9.png">

 - Template:
 - 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/61919617/203640048-d4f23798-cbbc-426d-a24d-5befbbea9515.png">